src/components/Theme/style.scss

```diff
@@ -6,40 +6,42 @@
   @include castor.Button();
 
   .ods-button {
-    border-radius: var(--button-border-radius);
+    border-radius: var(--osdk-border-radius-button);
 
     /* stylelint-disable-next-line selector-class-pattern */
     &.-action--primary {
-      background-color: var(--color-primary-button);
-      color: var(--color-primary-button-text);
+      background-color: var(--osdk-color-background-button-primary);
+      color: var(--osdk-color-content-button-primary-text);
+      border-color: var(--osdk-color-border-button-primary);
 
       &:active {
-        background-color: var(--color-primary-button-active);
+        background-color: var(--osdk-color-background-button-primary-active);
       }
 
       &:disabled {
-        background-color: var(--color-primary-button-disabled);
+        background-color: var(--color-background-button-primary-disabled);
+        border-color: $color-transparent;
       }
 
-      &:hover {
-        background-color: var(--color-primary-button-hover);
+      &:hover:not(:disabled) {
```

Comment:
No need for `:not()` here, simply move it before `&:disabled` and it will be overriden.